SECURITY.md

```diff
@@ -0,0 +1,239 @@
+1. Accept the Caliptra Embargo Policy that is outlined above.
+
+Removal:
+
+1. If a member stops adhering to these criteria after joining the list then the member will be unsubscribed.
+
+## Security incident management process
+
+Issues within this bug tracking system will transition through a number of states according to this diagram:
+
+<center>
+
+```mermaid
+flowchart TB
+    Triage --> Draft_New
+    Draft_New --> Draft_Assigned
+    Draft_New <--> Closed_Rejected
+    Draft_Assigned <--> Draft_Review
+    Draft_Review ---> Closed_Rejected
+    Draft_Review --> Draft_Accepted
+    Draft_Accepted --> Closed_Published
+```
+
+</center>
+
+- **Triage:** This state represents new reports that have been entered directly by a reporter. When entered by the security team in response to an
+email, the issue shall be transitioned directly to *Draft_New*.
+
+- **Draft_New:** This issue is awaiting Triage by the security team. The security team will analyze the issue, determine a responsible entity, assign
+it to that individual, and move the issue to the *Draft_Assigned* state. Part of triage will be to set the issue’s priority.
+
+- **Draft_Assigned:** The issue has been assigned, and is awaiting a fix by the assignee.
+
+- **Draft_Review:** Once there is a Caliptra pull request for the issue, the PR link will be added to a comment in the issue, and the issue moved to the
+*Draft_Review* state.
+
+- **Draft_Accepted:** Indicates that this issue has been merged into the appropriate branch within Caliptra.
+
+- **Closed_Published:** The embargo period has ended. The issue will be made publicly visible, the associated CVE updated, and the vulnerabilities page in the
+docs updated to include the detailed information.
+
+- **Closed_Rejected:** The Caliptra PSIRT has rejected the reported security vulnerability. It may have been deemed a non-issue, or it may have been converted to a standard Github Issue report.
+
+The security advisories created are kept private, due to the sensitive nature of security reports. The issues are only visible to certain parties:
+
+- Members of the PSIRT mailing list
+- The reporter
+- Others, as proposed and ratified by the Caliptra PSIRT. In the general case, this will include:
+  - The code owner responsible for the fix.
+  - The Caliptra release owners for the relevant releases affected by this vulnerability.
+
+The Caliptra TC shall review the reported vulnerabilities during any meeting with more than three people in attendance. During this review, they
+shall determine if new issues need to be embargoed.
+
+The guideline for embargo will be based on:
+
+1. Severity of the issue, and
+2. Exploitability of the issue.
+
+Issues that the Caliptra TC decides do not need an embargo will be reproduced in the regular Caliptra project bug tracking system.
+
+Security sensitive vulnerabilities shall be made public after an embargo period of at most 90 days. The intent is to allow 30 days within the
```

Comment:
90 days may be feasible for FW, but RTL / ROM may need longer.

Reply:
So if, for example, secure boot (RTL or ROM) is completely broken and Caliptra will run any firmware / malware, then that would be disclosed within 90 days?